.gitmodules

```diff
@@ -27,3 +27,6 @@
 [submodule "3rdparty/sentry"]
 	path = 3rdparty/sentry
 	url = https://github.com/getsentry/sentry-native/
+[submodule "3rdparty/swiftyping"]
```

Comment:
As we ship this with the client please update Licence.md :)

Reply:
Good call, added.